models/user.go

```diff
@@ -36,8 +36,8 @@ type User struct {
 	ResetToken          string      `json:"-"`
 	ReportsWeekly       bool        `json:"-" gorm:"default:false; type:bool"`
 	PublicLeaderboard   bool        `json:"-" gorm:"default:false; type:bool"`
-	SubscribedUntil     *CustomTime `json:"-" gorm:"type:timestamp" swaggertype:"string" format:"date" example:"2006-01-02 15:04:05.000"`
-	SubscriptionRenewal *CustomTime `json:"-" gorm:"type:timestamp" swaggertype:"string" format:"date" example:"2006-01-02 15:04:05.000"`
+	SubscribedUntil     *CustomTime `json:"-" gorm:"swaggertype:"string" format:"date" example:"2006-01-02 15:04:05.000"`
+	SubscriptionRenewal *CustomTime `json:"-" gorm:"swaggertype:"string" format:"date" example:"2006-01-02 15:04:05.000"`
```

Comment:
@ddadaal Just came across these two lines. The "swaggertype" thing looks odd there. I presume that's not the way how it's supposed to be?

Reply:
yes...it seems the `gorm:"` part should be removed

Reply:
:+1: Will do